### DISCUSSION TITLE
Next Release (>1.16) with [confirm] attribute - when?

### DISCUSSION THREAD

**Discussion by dream-dasher** [casey/just#1769] [Q&A] [answered]
Really looking forward to using the `[confirm]`attribute in some scripts.  (Especially to offer some heavier tool installs in shared repos.  To make access easier but keeping a hard opt-in before touching a package manager, for example.)

Currently `[confirm]` works if I clone the repo and build.  But isn't part of the public release in Brew or Cargo.  (1.16.0 was release in November)

Do we know when the release with confirm goes out?

**Comment by casey** [comment, marked as answer]
I just released 1.17.0, which includes the `[confim]` attribute. I don't know how long it will take for it to be available in Brew, but it's available in Cargo, and there are prebuilt binaries on the releases page. Enjoy!
